openedx/features/lti_course_tab/tab.py

```diff
@@ -114,7 +115,7 @@ def _get_context_title(course: CourseBlock) -> str:
         )
 
     def _get_lti_config(self, course: CourseBlock) -> LtiConfiguration:
-        raise NotImplementedError
+        return CourseLiveConfiguration.get(course.id).lti_configuration
```

Comment:
This method needs to be overridden in the child class. This base class is supposed to be generic